src/plays/tic-tac-toe/Game.jsx

```diff
@@ -0,0 +1,160 @@
+import React, { useState } from "react";
+import { toast, ToastContainer } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+
+import { FaTimes, FaRegCircle } from "react-icons/fa";
+import Icon from "./Icon";
+
+import "./Game.css";
+
+// Declaring an array to store the game state. Initializing it with 9 empty strings.
+const gameArray = new Array(9).fill("");
+
+const Game = () => {
+  let [isCross, setIsCross] = useState(true);
+  let [winMessage, setWinMessage] = useState("");

```

Comment:
You can have a draw, so it's not really a win message, it will be better to go with something like finalMessage or endgameMessage